packages/web/src/app/doc-pages/tools/utilities-doc/utilities-doc.component.html

```diff
@@ -10,34 +10,39 @@
 <div class="page-body">
   <!--TABLE OF CLASSES-->
   <app-component-subsection>
-    <div *ngFor="let group of utilityGroups">
```

Comment:
There seems to have been an extra div here, no visual difference when it is removed